protractor/e2e/integretion/second-page.spec.ts

```diff
@@ -55,9 +57,22 @@ describe('test for second page', () => {
         await secondPage.waitForElementVisible(secondPage.pdfContent);
     });
 
-    fit('info card error case', async () => {
+    it('info card error case', async () => {
         await secondPage.scrollToElement(secondPage.infoCardSection);
         await secondPage.waitForElementVisible(secondPage.infoEmailInpt);
         await secondPage.infoSubmitBtn.click();
     });
+
+    fit('should open page blog pages with articles', async () => {
```

Comment:
don't forget to remove "fit" from your tests before pushing it to the repository. Just imagine, that you have 1000 tests in your app and now in your CI will be only 1 test ran:-)

Reply:
Outdated code